addons/account_payment/views/account_portal_templates.xml

```diff
@@ -37,14 +57,87 @@
+                                        <t t-out="amount_custom or amount_next_installment" t-options="{'widget': 'monetary', 'display_currency': currency}"/>
+                                    </button>
+                                    <button class="btn btn-outline-primary o_btn_payment_tab"
+                                        id="o_payment_full_tab"
+                                        data-bs-toggle="pill"
+                                        data-bs-target="#o_payment_full"
+                                        type="button"
+                                        role="tab"
+                                        aria-controls="o_payment_full"
+                                        aria-selected="false"
+                                    >
+                                        <strong>Full Amount</strong><br/>
+                                        <t t-out="amount" t-options="{'widget': 'monetary', 'display_currency': currency}"/>
+                                    </button>
+                                </div>
+                                <div class="tab-content" id="o_payment_tabcontent">
+                                    <div class="tab-pane fade my-3 show active"
+                                        id="o_payment_installments"
+                                        role="tabpanel"
+                                        aria-labelledby="o_payment_installments_tab"
+                                        tabindex="0"
+                                    >
+                                        Payment Reference: <t t-out="invoice_name"/>-<t t-out="number_next_installment"/><br/>
+                                        Amount: <t t-out="amount_custom or amount_next_installment" t-options="{'widget': 'monetary', 'display_currency': currency}"/>
+                                    </div>
+                                    <div class="tab-pane fade my-3"
+                                        id="o_payment_full"
+                                        role="tabpanel"
+                                        aria-labelledby="o_payment_full_tab"
+                                        tabindex="0"
+                                    >
+                                        Payment Reference: <t t-out="invoice_name"/><br/>
+                                        Amount: <t t-out="amount" t-options="{'widget': 'monetary', 'display_currency': currency}"/>
+                                    </div>
+                                    <t t-call="account_payment.form"/>
+                                </div>
+                            </div>
+                            <t t-elif="amount > 0">
+                                <div class="mb-3">
+                                    Payment Reference: <t t-out="invoice_name"/><br/>
+                                    Amount: <t t-out="amount_custom or amount" t-options="{'widget': 'monetary', 'display_currency': currency}"/>
```

Comment:
Check if it wouldn't be better to use the `payment.summary_item` util template; we usually use it do display the summary of the payment context.

Reply:
Thanks for the tip! Looks better like that indeed. I'll check with our PO to make sure she's okay with it.

Reply:
Change was approved btw 👍

Reply:
You also have the single-tab part of the template to adapt.

Reply:
Actually it looks like the single-tab case is not considered at all anymore:
![image](https://github.com/user-attachments/assets/abcc15ed-c2b0-4108-a145-105dce434a18)

Reply:
Hmm, it works when I try it on runbot, but indeed I forgot to change the styling there.
How did you trigger this view?

Reply:
Updated the "no installments" case as well. Should be working now.